src/test/integration/uk/gov/hmcts/reform/sandl/snlevents/BaseIntegrationTest.java

```diff
@@ -3,16 +3,31 @@
 import org.junit.Before;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
 import org.springframework.test.context.junit4.SpringRunner;
 import uk.gov.hmcts.reform.sandl.snlevents.testcategories.IntegrationTest;
+import uk.gov.hmcts.reform.sandl.snlevents.testdata.builders.HearingPartBuilder;
+import uk.gov.hmcts.reform.sandl.snlevents.testdata.builders.SessionBuilder;
+import uk.gov.hmcts.reform.sandl.snlevents.testdata.builders.UpsertSessionBuilder;
 
 @RunWith(SpringRunner.class)
 @Category(IntegrationTest.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
     classes = Application.class)
+@Import(TestConfiguration.class)
 public abstract class BaseIntegrationTest {
 
+    @Autowired
+    public SessionBuilder sessionBuilder;
```

Comment:
Not sure why every integration test would have to have those builders, am I too DRY with this?

Reply:
You're right, but having it here clears up most of the test files that I have there right now. Also the SessionBuilder and HearingPartBuilder generate the fundamental entities in our system so I believe we will use them more in the future in majority of tests.